frontend/taipy-gui/base/src/index-preview.ts

```diff
@@ -0,0 +1,6 @@
+import { TaipyApp, createApp, OnChangeHandler, OnInitHandler } from "./app";
+import { ModuleData } from "./dataManager";
+
+export default TaipyApp;
+export { TaipyApp, createApp };
```

Comment:
are you exporting TaipyApp twice ?

Reply:
Yes it is right now. Thanks for spotting it. I am coming up with an update that allows chunking